front/app/modules/commercial/content_builder/admin/components/ContentBuilderTopBar/index.tsx

```diff
@@ -174,7 +175,7 @@ const ContentBuilderTopBar = ({
           disabled={disableSave}
           id="e2e-content-builder-topbar-save"
           buttonStyle="primary"
-          processing={loading}
+          processing={loading || isPendingState}
```

Comment:
I like that this shows a spinner on the Save button. Curious if the user might think all the data has been saved. We could disable it as an option but I'll leave that to you

Reply:
Sounds good - I'll set this on the "disabled" instead :) You're right that it could cause confusion otherwise.